src/QsFmt/Formatter.Tests/Formatter.Tests.fsproj

```diff
@@ -10,6 +10,7 @@
       <Link>TestCases\%(FileName)%(Extension)</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Compile Include="Tests.fs" />
```

Comment:
I'm a little surprised this is necessary... doesn't an fsproj automatically find any .fs files in the same directory tree?

Reply:
It is necessary, and F# projects don't automatically include files. F# files are compiled in the order they appear here in the project file (usually reflected in the solution explorer in the IDE). If you change the precedence of the files in an F# project, you can get different behavior, although usually you will just get errors. Since this order should be based on the logic and not on the alphabetical ordering of the file names, the order is not assumed from the directory structure, and so files can't be automatically included as it wouldn't know where to put the files in the order.